testsuites/benchmark/src/main/java/org/eclipse/rdf4j/benchmark/rio/ParserBenchmark.java

```diff
@@ -0,0 +1,90 @@
+/*******************************************************************************
+ * Copyright (c) 2021 Eclipse RDF4J contributors, Aduna, and others.
```

Comment:
Same comment as in the other PR: please change to "Copyright (c) 2021 Eclipse RDF4J contributors."

Sorry to be a bit pedantic about this, but for IP governance this is kinda important.

Reply:
Sorry, I thought I had just the year wrong, will fix it now :)

Reply:
FYI. The correct header is in the formatting section here: https://github.com/eclipse/rdf4j/blob/main/.github/CONTRIBUTING.md